server/grpc_bytestream.go

```diff
@@ -129,6 +129,24 @@ func (s *grpcServer) Read(req *bytestream.ReadRequest,
 	var chunkResp bytestream.ReadResponse
 	for {
 		n, err := rdr.Read(buf)
+
+		if n > 0 {
+			if limitedSend {
+				if (sendLimitRemaining - int64(n)) < 0 {
+					msg := "GRPC BYTESTREAM READ FAILED: READ LIMIT EXCEEDED"
+					return status.Error(codes.OutOfRange, msg)
+				}
+				sendLimitRemaining -= int64(n)
+			}
+
+			chunkResp.Data = buf[:n]
+			sendErr := resp.Send(&chunkResp)
+			if sendErr != nil {
+				s.accessLogger.Printf("GRPC BYTESTREAM READ FAILED TO SEND RESPONSE: %s", err)
```

Comment:
There's one last `err` here that needs to be changed to `sendErr`.

Reply:
Fixed